src/ir_Airton.h

```diff
@@ -68,7 +68,7 @@ const uint8_t kAirtonFanHigh = 0b100;  // 4
 const uint8_t kAirtonFanMax =  0b101;  // 5
 
 const uint8_t kAirtonMinTemp = 16;  // 16C
-const uint8_t kAirtonMaxTemp = 25;  // 25C
+const uint8_t kAirtonMaxTemp = 31;  // 31C
```

Comment:
Hey @crankyoldgit,
I am working on project that control Air conditioner by remote type from HomeKit with Homespan. 
Some one contacted me about one of AC that not passing 25C after checking and replacing this it passed and get till 31. I will try to get you samples but it will take me few days as I need contact this person.
Thanks!

Reply:
Thanks. Please do. I'm happy to wait.

Reply:
Hey @crankyoldgit Here is fully HEX from IR Transmitter: 
```
16
F28400003911D3
17
F18400013911D3
18
F08400023911D3
19
F78400033911D3
20
F68400043911D3
21
F58400053911D3
22
F48400063911D3
23
FB8400073911D3
24
FA8400083911D3
25
F98400093911D3
26
F884000A3911D3
27
FF84000B3911D3
28
FE84000C3911D3
29
FD84000D3911D3
30
FC84000E3911D3
31
E384000F3911D3

```
 Tried to make dump with this code:
```
airtonAc.setRaw(results.value);
uint8_t tempByte = (results.value >> 16) & 0xFF; 
Serial.print("Temperature byte in hex: ");
Serial.println(tempByte, HEX);
```
but without success

Reply:
@crankyoldgit do you need additional info ? 
Thanks!

Reply:
Nope. Looks good to me.